tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.extract.txt

```diff
@@ -1 +1,20 @@
 REGEXP_EXTRACT([str0], '(E.{3})')
+REGEXP_EXTRACT([key], "(key)") //"key"
```

Comment:
One thought for these new test cases. Do we want to structure them as new tests? `setup.string.regex.extract.extended.txt`, etc for example?

Reply:
@tabsdavis this is the right way to go, especially re those nagging failures we're seeing in proof runs. Tagging you to bump this to the top for you 👍